Fixed demo script

engelmi opened the pull request:
Relates to: https://github.com/containers/ramalama/pull/1876

Previously, no parameter was passed to the multi-model script function so that the shellcheck threw an error. In order to fix this, the directory of the script is detected and passed to assemble the firefox command with proper path.

@AlexonOliveiraRH Could you verify this change does not break the demo?

AlexonOliveiraRH commented:
> Relates to: #1876
> 
> Previously, no parameter was passed to the multi-model script function so that the shellcheck threw an error (since `dirname $1` was attemting to use it). In order to fix this, the directory of the script is detected and passed to assemble the firefox command with proper path.
> 
> @AlexonOliveiraRH Could you verify this change does not break the demo?

Relates to https://github.com/containers/ramalama/pull/1879

@engelmi, thank you for the heads up. I've just refactored the script to include some functionalities and fixed some other minor issues (check related PR). I guess this PR will become irrelevant then. Let's defer to the maintainers to check that.

engelmi commented:
> @engelmi, thank you for the heads up. I've just refactored the script to include some functionalities and fixed some other minor issues (check related PR). I guess this PR will become irrelevant then. Let's defer to the maintainers to check that.

Sure, lets close this PR in favor of #1879. I opened this PR only because the shellcheck failure introduced in #1876 causes the lint job of the CI to fail now. Please make sure to run `make lint` (or run shellcheck separately) @AlexonOliveiraRH

AlexonOliveiraRH commented:
> > @engelmi, thank you for the heads up. I've just refactored the script to include some functionalities and fixed some other minor issues (check related PR). I guess this PR will become irrelevant then. Let's defer to the maintainers to check that.
> 
> Sure, lets close this PR in favor of #1879. I opened this PR only because the shellcheck failure introduced in #1876 causes the lint job of the CI to fail now. Please make sure to run `make lint` (or run shellcheck separately) @AlexonOliveiraRH

@engelmi gotcha. Thank you for the heads up again and for continuing with the other PR. I'll take a look a the suggestions there.
